pxf/pxf-service/src/scripts/pxf-env.sh

```diff
@@ -41,6 +41,9 @@ fi
 # Port
 export PXF_PORT=${PXF_PORT:-51200}
 
+# Memory
+export PXF_HEAP_OPTS="-Xmx2g -Xms1g"
```

Comment:
I would call these PXF_JVM_OPTS since there can be other options here (like GC setting, etc)

Reply:
will change